### PR TITLE
AB#91 Gather more data from indexer & Remove duplicite NFTs

### PR DESCRIPTION
**Pull request by steboee**
#91
- add NFT property which are not NONE to neo4j
- push more information about NFT to  neo4j
- clean services from unused code
- BUGFIX - duplicite NFTs in neo4j fixed ✅

### CODE REVIEW

File: src/fetch_data.py [hunk 1]
```diff
@@ -12,21 +12,31 @@ def main():
...
                 amount
                 id
                 fromAddress
                 toAddress
                 nft {
-                id
+                    id
+                    name
+                    description
+                    uri
```
Comment (brano-hozza) [inline src/fetch_data.py:32]: Here you can also add attributes, its a map of features that NFT is made of, we can take those and also save them to DB. Later we will analyze them and create tags based on them

File: src/populate_neo.py [hunk 2]
```diff
@@ -29,11 +28,12 @@ def main():
...
             to_address=item['toAddress'],
             transaction_id=item['id'],
-            nft_id=item['nft']['id']
+            amount=item['amount'],
+            nft = item['nft']
         )
         counter += 1
         transactions.append(transaction)
-        if counter == 2000:
+        if counter == 1000:
```
Comment (brano-hozza) [inline src/populate_neo.py:36]: We could let user choose what size does he want

**Review by brano-hozza — CHANGES_REQUESTED** [review]

File: src/populate_neo.py (hunk 2 above)
Comment (steboee) [reply src/populate_neo.py]: ✅

File: src/fetch_data.py (hunk 1 above)
Comment (steboee) [reply src/fetch_data.py:31]: ✅

**Comment by steboee** [comment]
Nechajte ešte opened please

**Comment by steboee** [comment]
Okay, @brano-hozza  please review and merge

**Review by brano-hozza — APPROVED** [review]

**Pull request merged**